src/lib/utils.js

```diff
@@ -72,15 +72,10 @@ const images = importAll(requireContext('../assets/images', false, /\.(png|jpe?g
 const getTurkUniqueId = () => {
   const turkInfo = jsPsych.turk.turkInfo()
   const uniqueId = `${turkInfo.workerId}:${turkInfo.assignmentId}`
+  jsPsych.data.addProperties({turk_unique_id: uniqueId})
```

Comment:
the `turk_unique_id` is the participant id right? this should then be added as such